providers/cnr/CNRProvider.go

```diff
@@ -0,0 +1,81 @@
+// Package CNR implements a registrar that uses the CNR api to set name servers. It will self register it's providers when imported.
+package cnr
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/StackExchange/dnscontrol/v4/providers"
+	cnrcl "github.com/centralnicgroup-opensource/rtldev-middleware-go-sdk/v5/apiclient"
+)
+
+// GoReleaser: version
```

Comment:
I see this variable is used in the UserAgent. Should it be "dev"?  What are the other potential values?

Reply:
@tlimoncelli I can't follow regarding this point. cnrcl is an instance of our API Client Library that we use for API Communication. Comparable to the one in use by the HEXONET integration, just v5 which represents the new major version of that library just compatible with the CentralNic Reseller Provider API while v4 of this library works with the HEXONET Provider API only. Maybe you can go into more detail?

Reply:
Sorry for being unclear.  I'm actually referring to the "version" variable below this line.

Reply:
You mean Line 12? `// GoReleaser: version` ?

Reply:
Basically, it should be about the major version of DNSControl in use by the endcustomer so that we can access it on our end via API Request Logs / Command Logs. That sometimes helps us offering better support - we do that for all our integrations that way. By that there's no need to ask a customer which version of the software he has in use - we know that already and can act with it.

I have my doubts that it makes sense in the way we have it right now in place. If we execute `dnscontrol version` via shell, we receive `dnscontrol version dev`. This makes sense as this PR is wip. With a stable version in place like v3 or v4, I expect that we probably get a numeric string? We need exactly that particular information from DNSControl itself, but we have no idea how to access it. We could execute that shell command and parse the result, but maybe there's a more elegant programmatical way via DNSControl for achieving that? If you have an idea about the how, please let me know.